Doc/library/stdtypes.rst

```diff
@@ -1927,10 +1927,14 @@ expression support in the :mod:`re` module).
    ``{}``.  Each replacement field contains either the numeric index of a
    positional argument, or the name of a keyword argument.  Returns a copy of
    the string where each replacement field is replaced with the string value of
-   the corresponding argument.
+   the corresponding argument. For example:
+
+   .. doctest::
 
       >>> "The sum of 1 + 2 is {0}".format(1+2)
       'The sum of 1 + 2 is 3'
+      >>> "{1} expects the {0} Inquisition!".format("Spanish", "Nobody")
```

Comment:
nice addition! - would it be worth while adding an example with keyword arguments as well ? e.g `"The sum of 1 + 2 is {answer}".format(answer=1+2)`

Reply:
Done.